### ISSUE TITLE
CPU 97% [4] after installing theme

### ISSUE THREAD

**Issue by alundiak** [ctf0/Seti_ST3#136]
I did as it suggested in installation instructions (including a few restarts and files movement), but my cooler fan is like a rocket, and CPU is between 80-97%.
![](https://dl.dropboxusercontent.com/u/102154961/ST3.png)
![](https://dl.dropboxusercontent.com/u/102154961/ST3_2.png)

What this theme doing so heavy on my MacOS 10.10.3 and within ST3 (3083)

**Comment by ctf0** [comment]
am having the same config as u and everything is working fine, also i don't know why u have 4 copies running at the same time ???

**Comment by alundiak** [comment]
Yeah, I provided 2 different test cases, and I don't know also why I have 4 instances. I did clean install of Sublime yesterday, but not checked setup of Seti_UI again. Will do today.

**Comment by ctf0** [comment]
also if u work with large projects i would recommend disabling the file indexing from the user prefs.

**Comment by alundiak** [comment]
It was not large. 10 files. Now I'm going to setup theme again.

**Comment by alundiak** [comment]
Configured Seti_UI theme again, and issue reproduced 1 time (~90% CPU). But, I figured out, that it was indexing. So I disabled by `"index_files": false` in my User settings. **Hope now, it will never indexing and eat my CPU**.

But open questions till - why I have 4 instances of ST3 in processes list? Should I rise separate issue/bug? I no guess, it might be 4 threads, based on this description:

```
// Set the number threads to use for indexing. A value of 0 will make
// Sublime Text guess based on the number of cores. Use the index_files
// setting to disable all workers.
"index_workers": 0,
```

Because now, when everything runs properly, I have normal one process in list. 
If someone could confirm my guess, it will be enough to close this issue.
